packages/react-vapor/src/components/collapsible/CollapsibleContainerConnected.tsx

```diff
@@ -72,7 +72,7 @@ export const CollapsibleContainerDisconnected: React.FunctionComponent<
                 </div>
             }
             expandedOnMount={expandedOnMount}
-            headerClasses={classNames(styles.header, expanded ? 'bg-light-grey' : 'bg-white')}
+            headerClasses={classNames(styles.header)}

```

Comment:
We probably should have a class when it is expanded to be able to change the color in the scss

```suggestion
            headerClasses={classNames(styles.header, {'mod-expanded': expanded})}
```